docs/consume-packages/install-use-packages-visual-studio.md

```diff
@@ -14,155 +14,183 @@ f1_keywords:
+    ![Screenshot showing the Package source settings icon highlighted.](media/package-source-settings.png)
 
-1. Select the **Package Sources** node:
+1. In the **Options** window, expand the **NuGet Package Manager** node and select **Package Sources**.
 
-    ![Package Sources options](media/options.png)
+    ![Screenshot showing the Options window with Package Sources selected.](media/package-sources.png)
+
+1. To add a source, select **+**, edit the **Name**, enter the URL or path in **Source**, and then select  **Update**.
+
+   The source now appears in the **Package source** dropdown list.
 
-1. To add a source, select **+**, edit the name, enter the URL or path in the **Source** control, and select  **Update**. The source now appears in the selector drop-down.
 1. To change a package source, select it, make edits in the **Name** and **Source** boxes, and select **Update**.
-1. To disable a package source, clear the box to the left of the name in the list.
-1. To remove a package source, select it and then select the **X** button.
-1. Using the up and down arrow buttons does not change the priority order of the package sources. Visual Studio ignores the order of package sources, using the package from whichever source is first to respond to requests. For more information, see [Package restore](../consume-packages/package-restore.md).
 
-> [!Tip]
-> If a package source reappears after deleting it, it may be listed in a computer-level or user-level `NuGet.Config` files. See [Common NuGet configurations](../consume-packages/configuring-nuget-behavior.md) for the location of these files, then remove the source by editing the files manually or using the [nuget sources command](../reference/nuget-exe-CLI-reference.md).
+1. To disable a package source, clear the box to the left of the name in the list.
 
-## Package manager Options control
+1. To remove a package source, select it, and then select the **X** button.
 
-When a package is selected, the Package Manager UI displays a small, expandable **Options** control below the version selector (shown here both collapsed and expanded). Note that for some project types, only the **Show preview window** option is provided.
+   If a package source reappears after you delete it, it might be listed in a computer-level or user-level *NuGet.config* file. For the location of these files, see [Common NuGet configurations](../consume-packages/configuring-nuget-behavior.md). Remove the package source in the files by editing them manually or using the [nuget sources command](../reference/nuget-exe-CLI-reference.md).
 
-![Package manager options](media/PackageManagerUIOptions.png)
+Using the up and down arrow buttons doesn't change the priority order of the package sources. Visual Studio ignores the order of package sources, and uses the package from whichever source is the first to respond to a request. For more information, see [Restore packages](../consume-packages/package-restore.md).
```

Comment:
I removed these buttons in 17.2 Preview 1. https://github.com/NuGet/NuGet.Client/pull/4376